### ISSUE TITLE
Bad link to UoM Guide hosting on GH repo page

### ISSUE THREAD

**Issue by mikro2nd** [unitsofmeasurement/uom-guide#6]
The Github project page for the Guide at https://github.com/unitsofmeasurement/uom-guide points the unsuspecting and unwary noob to the published version of the guide at a link (https://www.gitbook.com/book/unitsofmeasurement/uom-guide/details) that frustratingly leads only to GitBooks insistent sales schpiel. For those of us way behind the curve on GitBook's unfriendly shenanigans, this leads, in turn, to a frustrating half-hour of searching around the Internet for the actual published content.

Might this link be updated to the correct one: https://unitsofmeasurement.gitbook.io/uom-guide/ thus saving the blood-pressure and time of millions of future users. ;)

Even better: might said content be hosted somewhere friendlier and more open than GitBook? I'm willing to assist if I can.

**Comment by keilw** [comment]
Judging from the downloads there are indeed Millions of users, but how many actual users would consume the guide I cannot say.
While Asciidoc was discussed and even given a try in other areas that does not seem a viable option or enthusiasticly picked up by "Millions" of eager editors either, so I would not go there unless you could demonstrate it offers the same reader experience especially with multi-language support like https://unitsofmeasurement.gitbook.io/uom-guide/advanced/advanced
(providing alternatives for Java or Kotlin, forget about the extent of the example, but I'm not aware of a quick and clean approach to that in Asciidoc)

We just migrated the TCK page in a similar way to a GH page, there are at least a dozen places, so let's keep an eye on it e.g. when doing something else like https://github.com/unitsofmeasurement/unit-tck-usage/issues/32.

Thanks for spotting it, but unless Gitbook also charges authors (if they charge a moderate fee for reading it so be it, but currently it seems free or up to authors to charge for those books) like some other self-publishing solutions, I don't think it's worth the effort and Asciidoc is not really a comparable solution especially when you browse the document online.

**Comment by mikro2nd** [comment]
Ja, ja. The alternative hosting suggestion was just an afterthought, really :) and not too seriously thought out. I agree that the cost/effort hardly justifies doing it. (Maybe.) The main thing is having a less circuitous way to find the content in the first place. :D
